qt/python/mantidqt/mantidqt/widgets/sliceviewer/test/test_sliceviewer_model.py

```diff
@@ -522,6 +538,37 @@ def test_calculate_axes_angles_uses_identity_if_W_unavailable(self):
         for iy in range(1, 3):
             self.assertAlmostEqual(axes_angles[0, iy], np.pi / 2, delta=1e-10)
 
+    def test_calculate_axes_angles_uses_identity_if_W_unavailable_MDHisto(self):
+        #test MD histo
+        ws = _create_mock_workspace(IMDHistoWorkspace,
+                                    SpecialCoordinateSystem.HKL,
+                                    has_oriented_lattice=True)
+        ws.getExperimentInfo().run().get.side_effect = KeyError
+        model = SliceViewerModel(ws)
+
+        axes_angles = model.get_axes_angles()
+        self.assertAlmostEqual(axes_angles[1, 2], np.pi / 2, delta=1e-10)
+        for iy in range(1, 3):
+            self.assertAlmostEqual(axes_angles[0, iy], np.pi / 2, delta=1e-10)
+
+    def test_calculate_axes_angles_uses_W_if_basis_vectors_unavailable_and_W_available_MDHisto(self):
+        #test MD histo
+        ws = _create_mock_workspace(IMDHistoWorkspace,
+                                    SpecialCoordinateSystem.HKL,
+                                    has_oriented_lattice=True,
+                                    ndims=3)
+        ws.getBasisVector.side_effect = lambda x: [0.0]
```

Comment:
Here you can just use `ws.getBasisVector.return_value = 0` - actually it's probably clearer to make it explicitly a 3-vector of zeros but this does work.

Reply:
That doesn't seem to work. It seems the `side_effect` still takes precedent

Reply:
Oops, fair enough!